.circleci/config.yml

```
@@ -343,3 +343,14 @@ workflows:
           environment: production-do-nyc3-c
           cluster: $K8S_PROD_NYC3_C
           subdomain: nyc3-c.prod.do
+
+      - deploy:
+          <<: *deploy-production
+          name: production-gcp-asia-southeast1-a
+          environment: production-gcp-asia-northeast1-a
```

```suggestion
          environment: production-gcp-asia-southeast1-a
```
same for the other variables below (s/northeast1/southeast1)